lib/src/users.rs

```diff
@@ -84,9 +87,9 @@ impl UserConfig {
         self.lookup_info(github_name).map(|u| u.slack_name)
     }
 
-    pub fn slack_direct_message(&self, github_name: &str) -> Option<SlackRecipient> {
+    pub fn slack_direct_message(&self, github_name: &str, repo_name: &String) -> Option<SlackRecipient> {
```

Comment:
i think we'd probably want this to be `org/repo`, no?

Reply:
That makes sense. I wasn't sure what exactly was on `.name` vs `.full_name`, but digging a little it looks like `.full_name` is `org/repo`. I made the change, although it's in messengers.rs. This is just the string param

Reply:
sounds good. maybe call it `repo` or `repo_full_name`? `repo_name` does look like it's the full name.

Reply:
Done, I went with `repo_full_name`